### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by PasVegan** [writer/writer-framework#132] [General]
Hi,
could you please keep the roadmap updated and make it more detailed because a lot of the change log in each version doesn't even appears on the roadmap.

Thanks !

**Comment by PasVegan** [comment]
That way if you do something good with linked issue/branch ect.. you could attract way more contributors because right now it's very hard to contribute because we don't know what you planned exactly and what is the state of each things.

**Comment by PasVegan** [comment]
Maybe you want some help to setup a proper AGILE environment ? I can help you with that if you want.

**Comment by ramedina86** [comment]
This is a fair request, there's growing interest (thankfully) so more clarity is needed. I'll work on this.

Most of the items in releases are a direct consequence of issues. So if you see open issues (especially bugs) those are going to be reflected in the next release.
